SeaPublicWebsite/Views/Error/ServiceIssue.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@{
+    ViewBag.Title = "Sorry, there is a problem with the service - Home Energy Retrofit Advice - GOV.UK";
+}
+
+@section BeforeMain {
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">Sorry, there is a problem with the service</h1>
+        <p class="govuk-body">
+            Try again later.
+        </p>
+        <p class="govuk-body">
+            We have not saved your answers. When the service is available, you will have to start again.
```

Comment:
This isn't always true?

Reply:
I thought about that. However if an error happens mid-journey (in the rare event of that), I don't think we should link the user back where he was left off. Based on [this](https://design-system.service.gov.uk/patterns/problem-with-the-service-pages/), asking for trying again is the most sensible.

Reply:
This is the design on the ticket, so fine here. We can raise separately if necessary